### PR TITLE
Update facets

### PR DESCRIPTION
**Pull request by sensei100**
Added the logic into traject for the library  availability facets.

### CODE REVIEW

File: spec/features/indices_spec.rb [hunk 1]
```diff
@@ -39,6 +39,7 @@
     context "publicly available pages" do
       scenario "User visits home page" do
         visit '/'
+        save_and_open_page
```
Comment (bibliotechy) [inline spec/features/indices_spec.rb]: @sensei100 Is this an artifact of testing?

File: app/views/catalog/_show_default.html.erb [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <div class="availability">
     <dl class="dl-horizontal  dl-invert">
       <dt class="blacklight-availability show-page-avail">Status/Location:</dt>
+      <span id="alma_availability_url" data-url="<%= alma_availability_url(format: :json) %>" ></span>
```
Comment (bibliotechy) [inline app/views/catalog/_show_default.html.erb]: Is this part of this commit?

Comment (bibliotechy) [reply app/views/catalog/_show_default.html.erb]: Oh, never mind, I remember now, this is going on the show page

Comment (sensei100) [reply app/views/catalog/_show_default.html.erb]: Yeah, sorry about that. I probably should have put it in a different branch.

**Pull request merged**